Makefile.vendor.mk

```diff
@@ -0,0 +1,6 @@
+VERSION = 3.0.0
```

Comment:
Do we want to use '3.0.0-tp.1' here or overwrite this in cpaas midstream?

Reply:
same here, I'd say we do that when we branch off into a release branch for tp1